server/src/api/board_sessions.go

```diff
@@ -101,3 +101,67 @@ func (s *Server) updateBoardSessions(w http.ResponseWriter, r *http.Request) {
 	render.Status(r, http.StatusOK)
 	render.Respond(w, r, sessions)
 }
+
+func (s *Server) resetAllReadyStates(w http.ResponseWriter, r *http.Request) {
+	var desiredUserState = false
+	log := logger.FromRequest(r)
+	currentBoard := r.Context().Value("Board").(uuid.UUID)
+	caller := r.Context().Value("User").(uuid.UUID)
+
+	filter := database.BoardSessionFilterTypeFromQueryString(r.URL.Query())
+	sessions, err := s.sessions.List(r.Context(), currentBoard, filter)
+	if err != nil {
+		log.Errorw("unable to get board sessions", "err", err)
+		common.Throw(w, r, common.InternalServerError)
+	}
+
+	for _, session := range sessions {
+		board := currentBoard
+		caller := caller
```

Comment:
not sure if these assignments are necessary :)

Reply:
Done. :)